ui/app/templates/components/oidc/scope-form.hbs

```diff
@@ -78,40 +80,25 @@
   {{/if}}
 </form>
 
-<Modal
-  @title="Scope template"
-  @onClose={{fn (mut this.showTemplateModal) false}}
-  @isActive={{this.showTemplateModal}}
-  @showCloseButton={{true}}
->
-  <section class="modal-card-body">
-    <div class="is-flex-between is-flex-center has-bottom-margin-s">
+{{#if this.showTemplateModal}}
+  <Hds::Modal id="scope-template-modal" @size="large" @onClose={{fn (mut this.showTemplateModal) false}} as |M|>
+    <M.Header>
+      Scope template
+    </M.Header>
+    <M.Body>
       <p data-test-modal-text>
         Example of a JSON template for scopes:
       </p>
-      <Hds::Copy::Button
```

Comment:
Removed this copy button and removed `@showToolbar={{false}}` from the JsonEditor to align with elsewhere we use the JsonEditor to render sample templates